### PR TITLE
Add sticky Header

### LINKED ISSUE #285: Feature: Sticky navigation menu
## Requirements
- Navigation menu should be sticky and always visible when scrolling

### PR DESCRIPTION
**Pull request by AgentRBY**
## Description
Add sticky Header

## Motivation and Context
Fixes #285

## Screenshots (if appropriate):
image

### CODE REVIEW

File: src/layout/Navbar/Navbar.scss [hunk 1]
```diff
@@ -7,7 +7,8 @@
 }
 
 .navbar {
-	position: relative;
+	position: sticky;
+	top: 0;
```
Comment (TheOnlyTails) [inline src/layout/Navbar/Navbar.scss:11]: use the logical property:
```suggestion
	inset-block-start: 0;
```

**Comment by yaira2** [comment]
Should there be a border when it's sticky?

**Comment by AgentRBY** [comment]
> Should there be a border when it's sticky?

Border?

**Comment by yaira2** [comment]
A divider line on the bottom, alternatively we can blur the nav bar background when scrolling like we do on mobile.

**Comment by AgentRBY** [comment]
I added blur like on mobile
image

**Review by yaira2 — DISMISSED** [review]
LGTM

File: src/layout/Navbar/Navbar.scss [hunk 2]
```diff
@@ -7,13 +7,15 @@
...
+	position: sticky;
+	inset-block-start: 0;
 	z-index: 10;
 	justify-content: space-between;
 	flex: 0 0 auto;
 	block-size: 58px;
 	padding: 12px 18px;
-	background-color: var(--fds-solid-background-base);
+	background-color: hsl(var(--mica-tint), var(--mica-tint-opacity));
+	backdrop-filter: blur(60px) saturate(150%);
```
Comment (chingucoding) [inline src/layout/Navbar/Navbar.scss:18]: @yaira2 @TheOnlyTails What are your thoughts on reducing the blur radius? Right now, when scrolling over the images it flashes quite a lot, even when scrolling just a few pixels. I think something in the range of 20px would be better.

Comment (yaira2) [reply src/layout/Navbar/Navbar.scss:18]: I don't see flashes but could depend on the device, probably a good idea to reduce it.

**Comment by AgentRBY** [comment]
Okay, something's broken.

**Comment by AgentRBY** [comment]
Fixed

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**